_posts/2024-07-22-pyopensci-at-scipy-2024.md

```diff
@@ -0,0 +1,463 @@
+were smaller in size and scope. Over the years, SciPy has evolved significantly,
+mirroring the [🚀 explosive growth of the Python ecosystem 🚀](https://stackoverflow.blog/2023/01/26/comparing-tag-trends-with-our-most-loved-programming-languages/)
+in scientific computing and data science.
+
+Today, the SciPy Conference, hosted by [NumFocus](https://numfocus.org/),
+attracts over 700 attendees from diverse scientific fields such as astronomy,
+biology, geophysics, and more. SciPy is much more than just a workshop.
+
+SciPy features:
+
+* Incredible keynote speakers, such as the inspirational
+  [Carol Willing](https://www.youtube.com/watch?v=s-W-UvGgDco).
+* Talks from the community about tools and approaches.
+* Poster sessions that allow presenters to directly engage and discuss cool
+  work with the meeting attendees.
+* Tutorials led by community members (and pyOpenSci) that cover important
+  scientific data processing and analysis-related Python skills.
+* Sprints: collaborative coding sessions where people come to contribute to
+  and learn about open-source projects.
+
+I'd be remiss if I didn't mention the lightning talk sessions. During lightning
+talks, people throw their names into a literal bucket. If their name is selected,
+they are given 5 minutes (or less if the dice are rolled) to talk about a topic
+they love. At SciPy, there are often off-beat distractions from the moderators
+along the way, such as the infamous grab claw (see below). Yes, it is as
+ridiculously hilarious as it sounds.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/july/scipy-24-lobster-claw.jpeg" alt="Photograph showing a man with a beard holding a red lobster claw - while deviously looking at a blonde woman at a podium who is smirking at him. She is wearing a blue top and beige pants" style="width: 90%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> It's well-known that lightning talk organizers enjoy heckling presenters. Sometimes they do so using props, such as the lobster claw seen here <i class="fa-solid fa-heart-pulse"></i>. Photo Credit: SciPyConf Fosstodon </figcaption>
+</figure>
+
+[Did someone mention sharks? More here on my experience last year giving a SciPy lighting talk.](scipy-2023-community.html)
+
+The entire meeting is community-driven. The scientific Python community really
+shows up to make this meeting what it is. All of the sessions are volunteer
+organized and run. Volunteers also spend considerable time engaging with
+presenters, ensuring there are social activities that everyone is invited to in
+the evenings, and generally making sure that everyone feels included.
+
+Since the pandemic, the SciPy meeting has adapted to support virtual/hybrid
+participation, which has further increased its reach. In fact, some remote
+attendees also helped organize the meeting!
+
+If you are a Pythonista who loves science, this meeting might be for you!
```

Comment:
```suggestion
If you are a Pythonista who loves science, this meeting is for you!
```

given that your post has been very complimentary of the event, I think it's OK to make a clear, declarative statement about attending.